deeppixel/iqa/Test.py

```diff
@@ -0,0 +1,21 @@
+import brisque as b
```

Comment:
Can you please shift this to a test directory at the root level. Since inside the package, we don't require this 🙂

Reply:
Ok then.